sdk/src/main/scala/com/stratio/sparkta/sdk/Output.scala

```diff
@@ -172,8 +182,11 @@ abstract class Output(keyName: String,
 
 object Output {
 
-  final val SEPARATOR = "_"
-  final val ID = "id"
+  final val Separator = "_"
+  final val Id = "id"
+  final val FixedAggregation = "fixedAggregation"
```

Comment:
We need change all multiplexer operations to rollup level as we discuss the last week.
This three constants, where do you think that they must be declared?